### DISCUSSION TITLE
https://notes.alexkehayias.com/emacs-in-the-browser-using-codespaces/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#4] [Announcements]
# https://notes.alexkehayias.com/emacs-in-the-browser-using-codespaces/

Using codespaces, you can open a web-based VSCode session, open a terminal and install emacs.

https://notes.alexkehayias.com/emacs-in-the-browser-using-codespaces/

**Comment by henrik-ch** [comment]
This is great. Even better than the web-based version imho, is gh codespace ssh and have emacs available on the terminal.
